tests/next_tests/integration_tests/cases.py

```diff
@@ -492,6 +492,21 @@ def cartesian_case(
     )
 
 
+@pytest.fixture
+def cartesian_case_no_backend():
```

Comment:
The `cartesian_case` fixture could use this one to avoid code duplication.

Reply:
I still like it better this way since the original fixture is parametrized with all the backends and here we only want to run it once and using a custom backend.

Reply:
Refactored as discussed offline.